### PR TITLE
Delete code under !NO_CRT define

### PR DESCRIPTION
**Pull request by jkotas**
It is always defined - even in internal builds

### CODE REVIEW

File: src/utilcode/stgpool.cpp [hunk 1]
```diff
@@ -361,7 +361,7 @@ bool StgPool::Grow(         // true if successful.
         return true;
     }
 
-#ifndef NO_CRT
+#if 0
```
Comment (jkotas) [inline src/utilcode/stgpool.cpp]: `#if 0` to match the existing behavior. It is unclear whether the code under this ifdef is good.

**Comment by jkotas** [comment]
@janvorli PTLA

File: src/utilcode/debug.cpp [hunk 2]
```diff
@@ -622,8 +622,6 @@ unsigned DbgGetEXETimeStamp()
 // // //  Don't call this with a K larger than 159.
 // // //
 
-#if !defined(NO_CRT)
```
Comment (janvorli) [inline src/utilcode/debug.cpp]: I am a bit confused - the NO_CRT was defined before and so the code under this `#if` was excluded. Now we have got rid of the `#if`, but we include the previously excluded stuff here.

File: src/utilcode/stgpool.cpp (hunk 1 above)
Comment (janvorli) [inline src/utilcode/stgpool.cpp]: Should we just remove the code instead?

**Comment by janvorli** [comment]
LGTM modulo the comments.

File: src/utilcode/stgpool.cpp (hunk 1 above)
Comment (jkotas) [inline src/utilcode/stgpool.cpp]: Deleted.

File: src/utilcode/debug.cpp (hunk 2 above)
Comment (jkotas) [inline src/utilcode/debug.cpp]: Deleted, as well as a comment in utilcode.h that referred to it.

**Pull request merged**